### PR TITLE
136 fix mobile layout

### PR DESCRIPTION
**Pull request by johnxussw**
#136 

TODO:
- [x] Consulting page - responsive top module
- [x] Consulting page - responsive testimonials
- [x] Consulting page - responsive booking buttons
- [x] Centered companies' logo
- [x] Megamenu alignment
- [x] Upgrade legacy/image in clientLogo component
- [x] Component | Booking button | Booking form - responsive

image
**Figure: responsive top module**

image
**Figure: responsive testimonials**

image
**Figure: responsive booking button 1**
image
**Figure: responsive booking button 2**
image
**Figure: responsive booking button 3**

image
**Figure: centered companies' logo**

image
**Figure: Megamenu alignment**

image
**Figure: Booking form on mobile**

### CODE REVIEW

File: components/blocks/booking.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const Booking: FC<{
 }> = ({ title, subTitle, videoBackground, children }) => {
   return (
     <>
-      <Container padding="px-4">
+      <Container padding="px-4 w-full">
```
Comment (wicksipedia) [inline components/blocks/booking.tsx:13]: `w-full` isn't a padding size

**Review by wicksipedia — CHANGES_REQUESTED** [review]

File: components/blocks/booking.tsx (hunk 1 above)
Comment (johnxussw) [reply components/blocks/booking.tsx]: Done
Change from
```
<Container padding="px-4 w-full">
```
to
```
<Container padding="px-4" className="w-full">
```

**Review by wicksipedia — CHANGES_REQUESTED** [review]
The booking form is too big on a phone - can it be resized for phone

image

**Review by wicksipedia — DISMISSED** [review]
lgtm

**Review by wicksipedia — APPROVED** [review]
Lgtm

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**